src/Components/MyLists/MyLists.tsx

```diff
@@ -8,46 +8,68 @@ interface Props {
   removeFromList: (listName: string, id: string) => void;
 }
 
-// pass removeFromList to card components
-
-const MyLists: React.FC<Props> = (props: Props) => {
-  const createCurrentList = (listName: string) => {
-    return props.userLists[listName].map(restaurant => { return (
-      <ListCard 
-      key={restaurant.id}
-      id={restaurant.id}
-      removeFromList={props.removeFromList}
-      name={restaurant.name}
-      rating={restaurant.rating}
-      image={restaurant.image_url}
-      location={restaurant.location.display_address}
-      phone={restaurant.phone}
-      url={restaurant.url}
-      />
-    )
-  })
+interface State {
+  cardsToDisplay: JSX.Element[], // need to double check how to type this
```

Comment:
@hannahhch is `JSX.Element[]` an TS appropriate way to assign type? Or do we need to create an interface specifically for the cardToDisplay.

Reply:
This works fine! You'd only make an interface if there was something extra you wanted to require here.